src/linter.ts

```diff
@@ -1,7 +1,6 @@
 import { DiagnosticSeverity, JsonPath } from '@stoplight/types';
 import * as jp from 'jsonpath';
-import get = require('lodash/get');
-import has = require('lodash/has');
+import { get, has } from 'lodash';
```

Comment:
Does this pass linting? In most of our other projects imports must be sorted alphabetically.

Reply:
Yeah I think so, since 'g' is placed before 'h' in the alphabet and is 'l' comes after 'j'.